Letterbook.Workers/Publishers/ActivityMessagePublisher.cs

```diff
@@ -0,0 +1,56 @@
+using ActivityPub.Types.AS;
+using ActivityPub.Types.Conversion;
+using Letterbook.Core;
+using Letterbook.Core.Adapters;
+using Letterbook.Core.Extensions;
+using Letterbook.Core.Models;
+using Letterbook.Workers.Contracts;
+using MassTransit;
+using Microsoft.Extensions.Options;
+
+namespace Letterbook.Workers.Publishers;
```

Comment:
Similar, the MassTransit publishers are updates of the event* services that were previously in core. They even still implement the same interfaces.